src/main/webapp/app/shared/util/tab.constants.ts

```diff
@@ -45,3 +45,26 @@ export const eventTabList = (eventId: string): ITabInfo[] => [
     tabRoute: `/entity/event-crew/event/${eventId}`
   }
 ];
+
+export const financeTabList: ITabInfo[] = [
+  {
+    tabName: 'Report',
+    tabTranslateKey: 'clubmanagementApp.event.upcoming',
```

Comment:
what is upcoming?

Reply:
Corrected it in https://github.com/Five-Fishes/Club-Management-React-Client/pull/78/commits/8a0dac9be11658f44681f25535fbbbd24578f440